GITHUB_APP_GUIDE.md

```diff
@@ -0,0 +1,81 @@
+# Create GitHub App
+
+### Required CLI tools
+  - `smallstep` [link](https://smallstep.com/docs/step-cli/installation)
+  - `jq` [link](https://stedolan.github.io/jq/download/)
+  - `curl` 
+
+## Create GitHub App
+
+1. Go to your profile / organization settings by clicking to the link below:
+
+   - Organization https://github.com/organizations/ORG_NAME/settings/apps
+     - Replace `ORG_NAME` with the name of your organization
+   - Personal https://github.com/settings/apps
+
+2. Click to `New GitHub App`
+3. Set the name for your GitHub App (e.g. push-bot)
+4. As we don't host your app set `http://example.com` for `Homepage URL`
+5. Disable `Webhook` by unchecking `Active` button
+6. Change following permissions
```

Comment:
Suggestion: screenshot of the properly-configured permissions page here would be nice.

Reply:
I must say I agree with this, but this page is 50+ options and it's about 4-5 scrolling to get to the end
as you pointed to this specific part I've added additional info for all other options
update: also i found some syntax/typo issues which are fixed now